library/build.gradle

```diff
@@ -47,7 +47,7 @@ publishing {
 dependencies {
     api 'org.freemarker:freemarker:2.3.28'
     api 'org.jboss.resteasy:resteasy-jaxrs:3.6.1.Final'
-    api 'javax.servlet:javax.servlet-api:4.0.1'
+    api 'jakarta.servlet:jakarta.servlet-api:6.0.0'
     api 'org.apache.logging.log4j:log4j-core:2.17.0'
     api 'org.jboss.resteasy:resteasy-jackson2-provider:3.6.1.Final'
```

Comment:
Does RESTEasy need to be converted to Jakarta as well?

Reply:
Done